docs/develop/dotnet/asynchronous-activity.mdx

```diff
@@ -22,11 +22,9 @@ keywords:
   - completeasyncexception
   - getasyncactivityhandle
 tags:
-  - asynchronous-activity
-  - overview
-  - sdk
-  - dotnet-sdk
-  - developer guide
+  - Activities
+  - .Net SDK
+  - Temporal SDKs
```

Comment:
Is "Temporal SDKs" a useful tag for users?

Reply:
It would certainly get a lot of results, I'd imagine

Reply:
I am leaning toward dropping it but wasn't sure.

Reply:
Leaving for now and will measure.